src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+3.4.0
+-----
+
+* Deprecated not using the new `KERNEL_CLASS` server var with `KernelTestCase`.
```

Comment:
Not only. What is deprecated is the whole current method implementation, i.e KERNEL_DIR and the auto-detection based on phpunit.xml dir location. <=> Not using `KERNEL_CLASS`